tools/interactive/interactivetool_radiant.xml

```diff
@@ -0,0 +1,75 @@
+<tool id="interactive_tool_radiant" tool_type="interactive" name="radiant" version="0.1">
+    <description>Data analytics using Radiant R Shiny app</description>
+    <requirements>
+        <container type="docker">ylebras/radiant-docker</container>
+    </requirements>
+    <entry_points>
+        <entry_point name="radiant visualisation" requires_domain="True">
+            <port>3838</port>
+            <url>/sample-apps/STAT/</url>
+        </entry_point>
+    </entry_points>
+    <environment_variables>
+
+        <environment_variable name="HISTORY_ID" strip="True">${__app__.security.encode_id($outfile.history_id)}</environment_variable> <!-- FIXME: Warning: The use of __app__ is deprecated and will break backward compatibility in the near future -->
```

Comment:
please adopt this to use the new schema from jupyter.

Reply:
done